spec/unit/pops/types/type_calculator_spec.rb

```diff
@@ -208,6 +208,12 @@ def types
       end
     end
 
+    context 'sensitive' do
+      it 'translates to PSensitiveType' do
+        expect(calculator.infer(PSensitiveType::Sensitive.new("hunter2")).class).to eq(PSensitiveType)
+      end
+    end
```

Comment:
We actually need to expect the opposite to prevent information leakage - exposing the fact that this is a 6 character string leaks a huge amount of information. If this was a password this information reduces the brute force attack space of this password enormously. I've pushed another commit to enforce generalization of the contained type; I think this needs more thinking.